### PR TITLE
Frontend config and Loader component

### PR DESCRIPTION
**Pull request by mrdrogdrog**

### CODE REVIEW

File: src/components/loader/application-loader.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  .animation-pulse {
+    animation: pulse 2s ease-in-out infinite;
+  }
+
+  .animation-shake {
+    animation: shake 0.3s ease-in-out;
+  }
+
+  .icon {
+
```
Comment (DerMolly) [inline src/components/loader/application-loader.scss:12]:
```suggestion
```

File: src/redux/backend-config/actions.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import {Action, ActionCreator} from 'redux';
+import {BackendConfigState} from "./types";
+
+export const SET_BACKEND_CONFIG_ACTION_TYPE = 'config/set';
```
Comment (DerMolly) [inline src/redux/backend-config/actions.ts:4]:
```suggestion
export const SET_BACKEND_CONFIG_ACTION_TYPE = 'backend-config/set';
```

File: src/redux/frontend-config/actions.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {Action, ActionCreator} from 'redux';
+import {FrontendConfigState} from "./types";
+
+export const SET_FRONTEND_CONFIG_ACTION_TYPE = 'config/set';
```
Comment (DerMolly) [inline src/redux/frontend-config/actions.ts:4]:
```suggestion
export const SET_FRONTEND_CONFIG_ACTION_TYPE = 'frontend-config/set';
```

File: public/config [hunk 4]
```diff
@@ -1,20 +1,3 @@
...
-    "email": false
-  },
-  "specialLinks": {
-    "privacy": "test",
-    "termsOfUse": "test",
-    "imprint": "test"
-  }
-}
+"backendUrl": "http://localhost:3000"
+}
```
Comment (SuperSandro2000) [inline public/config:3]:
```suggestion
}

```

File: src/redux/backend-config/actions.ts (hunk 2 above)
Comment (mrdrogdrog) [reply src/redux/backend-config/actions.ts]: Renamed

File: src/components/loader/application-loader.scss (hunk 1 above)
Comment (mrdrogdrog) [reply src/components/loader/application-loader.scss]: done

File: src/redux/frontend-config/actions.ts (hunk 3 above)
Comment (mrdrogdrog) [reply src/redux/frontend-config/actions.ts]: Renamed

File: public/config (hunk 4 above)
Comment (mrdrogdrog) [reply public/config]: done

File: src/redux/backend-config/types.ts [hunk 5]
```diff
@@ -1,10 +1,13 @@
-export interface ApplicationConfigState {
+import {Action} from "redux";
+
+export const SET_BACKEND_CONFIG_ACTION_TYPE = 'frontend-config/set';
```
Comment (DerMolly) [inline src/redux/backend-config/types.ts:3]:
```suggestion
export const SET_BACKEND_CONFIG_ACTION_TYPE = 'backend-config/set';
```

**Pull request merged**